pyg_lib/ops/__init__.py

```diff
@@ -32,12 +32,12 @@ def grouped_matmul(inputs: List[Tensor], others: List[Tensor]) -> List[Tensor]:
         List[torch.Tensor]: List of 2D output matrices of shapes
         :obj:`[N_i, M_i]`.
     """
-    outs = torch.ops.pyg.grouped_matmul(inputs, others)
-
-    for src, other, out in zip(inputs, others, outs):
-        out.requires_grad = src.requires_grad or other.requires_grad
-
-    return outs
+    major_vers, minor_vers = str(torch.__version__).split('.')[:2]
+    assert int(major_vers) >= 2 or int(minor_vers) >= 14, (
```

Comment:
torch.bmm is only working on nested tensors in >= 1.14, this is needed to implement grouped matmul since pytorch does no support autograd for lists of tensors so we would never be able toget the backwards implementation working

Reply:
Ok, but what's the problem with that? I think this function should still work for < 1.14. So either we dispatch `ops.pyg.grouped_matmul` as before, or apply different computation dependent on PyTorch versioning.